nmostesting/suites/IS0703Test.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (C) 2019 Advanced Media Workflow Association
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ..GenericTest import GenericTest
+from ..IS05Utils import IS05Utils
+from ..IS07Utils import IS07Utils
+
+EVENTS_API_KEY = "events"
+CONN_API_KEY = "connection"
+
+
+class IS0703Test(GenericTest):
+    """
+    Runs IS-07-03-Test
+    """
+    def __init__(self, apis):
+        GenericTest.__init__(self, apis)
+        self.events_url = self.apis[EVENTS_API_KEY]["url"]
+        self.connection_url = self.apis[CONN_API_KEY]["url"]
+        self.is05_utils = IS05Utils(self.connection_url)
+        self.is07_utils = IS07Utils(self.events_url)
+
+    def set_up_tests(self):
+        self.senders = self.is05_utils.get_senders()
+        self.sources = self.is07_utils.get_sources_states_and_types()
+        self.transport_types = {}
+        self.sender_active_params = {}
+
+        for sender in self.senders:
+            if self.is05_utils.compare_api_version(self.apis[CONN_API_KEY]["version"], "v1.1") >= 0:
+                self.transport_types[sender] = self.is05_utils.get_transporttype(sender, "sender")
+            else:
+                self.transport_types[sender] = "urn:x-nmos:transport:rtp"
+
+        if len(self.senders) > 0:
+            for sender in self.senders:
+                dest = "single/senders/" + sender + "/active/"
+                valid, response = self.is05_utils.checkCleanRequestJSON("GET", dest)
+                if valid:
+                    if len(response) > 0 and isinstance(response["transport_params"][0], dict):
```

Comment:
I guess we don't want an exception at this point, so all the extra checks are needed... hmmm.